.github/workflows/ci.yml

```diff
@@ -182,7 +182,7 @@ jobs:
 
       - name: Standard Tests
         if: startsWith(runner.os, 'Windows') == false &&  matrix.skeleton == false
-        run: cmake --build ${{ matrix.build-dir }} --target check
+        run: cmake --build ${{ matrix.build-dir }} --target check --verbose
```

Comment:
This is an unrelated change. Not necessary for this PR, but it would be better to do it in a separate commit.

Reply:
Good catch!